tools/auto_fill_inference_cache.py

```diff
@@ -150,29 +258,40 @@ def compile_and_cache_model(hf_model_id, batch_size, sequence_length, num_cores,
                 config = json.load(f)
         for model_id, configs in config.items():
             for model_config in configs:
-
                 compile_and_cache_model(
                     hf_model_id=model_id,
+                    inference_type=model_config["inference_type"],
```

Comment:
This field does not exist in the model configs, and I think it will be a bit tedious to add it every single time. It should be deduced from the model_id IMHO.

See for instance the config for llama variants:

https://huggingface.co/aws-neuron/optimum-neuron-cache/blob/main/inference-cache-config/llama-variants.json

Reply:
So basically, what I am requesting is that the script continues to accept the existing config files with minimal changes.